warehouse/query-core/src/test/java/datawave/query/predicate/TLDEventDataFilterTest.java

```diff
@@ -497,7 +495,7 @@ public void functional_ContentExpansionTest() throws ParseException {
         ASTJexlScript newScript = FunctionIndexQueryExpansionVisitor.expandFunctions(config, helper, helper2, script);
 
         Map<String,ExpressionFilter> expressionFilters = getExpressionFilters(newScript, new AttributeFactory(mockTypeMetadata));
-        filter = new TLDEventDataFilter(newScript, Sets.newHashSet("FOO", "BAR"), expressionFilters, null, null, -1, 1);
+        filter = new TLDEventDataFilter(newScript, new HashSet<>(Arrays.asList("FOO", "BAR")), expressionFilters, null, null, -1, 1);
```

Comment:
I would have left the `Sets.newHashSet()` calls

Reply:
guava says to use the regular java constructor and that "newHashSet()" will likely be deprecated in the future